src/Task1/Person.java

```diff
@@ -0,0 +1,69 @@
+package Task1;
+
+public class Person {
+    private String firstName;
+    private String lastName;
+    private int age;
+    private boolean isActive;
+
+    public Person(final String firstName, final String lastName, final boolean isActive) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.isActive = isActive;
+    }
```

Comment:
This constructor should have called the one with 4 parameter and a default value for age.
i.e.:
`this(firstName, lastName, 0, isActive)`.
The way you did it, `this.age` is never assigned a value. This is possible, because `int` in Java is always initialized to 0, but it is advisable to always have some default value to avoid undefined or unexpected behaviour down the line, especially with `null`.